internal/services/cdn/cdn_frontdoor_profile_resource_test.go

```diff
@@ -85,6 +127,97 @@ func TestAccCdnFrontDoorProfile_update(t *testing.T) {
 	})
 }
 
+func TestAccCdnFrontDoorProfileWithSystemIdentity_update(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_cdn_frontdoor_profile", "test")
+	r := CdnFrontDoorProfileResource{}
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.complete(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+				check.That(data.ResourceName).Key("response_timeout_seconds").HasValue("240"),
+			),
+		},
+		data.ImportStep(),
+		{
+			Config: r.updateWithSystemIdentity(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+				check.That(data.ResourceName).Key("response_timeout_seconds").HasValue("120"),
+			),
+		},
+		data.ImportStep(),
+		{
+			Config: r.complete(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+				check.That(data.ResourceName).Key("response_timeout_seconds").HasValue("240"),
+			),
+		},
+	})
```

Comment:
Missed import step here?
```suggestion
		data.ImportStep(),
		{
			Config: r.complete(data),
			Check: acceptance.ComposeTestCheckFunc(
				check.That(data.ResourceName).ExistsInAzure(r),
				check.That(data.ResourceName).Key("response_timeout_seconds").HasValue("240"),
			),
		},
		data.ImportStep(),
	})
```

Reply:
Fixed.